### PR TITLE
Update api.md: timestamp format when measure

### PR DESCRIPTION
**Pull request by AlvaroVega**

### CODE REVIEW

**Comment by fgalan** [comment]
Let's keep this PR on hold until the behaviour on how manage TimeInstant get clarified.

**Comment by AlvaroVega** [comment]
Maybe we should add the follwing table to doc:

timeInstant value | measure contains timeInstant | actual behaviour 
-- | -- | -- 
true | Yes | TimeInstant and metadata updated with measure value 
true | No | TimeInstant and metadata updated with server timestamp 
false | Yes | TimeInstant and metadata updated with measure value 
false | No | TimeInstant and metadata updated with server timestamp 
Not defined | Yes | TimeInstant and metadata updated with measure value 
Not defined | No | TimeInstant and metadata updated with server timestamp

File: doc/api.md [hunk 1]
```diff
@@ -955,6 +955,19 @@ The IOTA processes the entity attributes looking for a `TimeInstant` attribute.
 adds a `TimeInstant` attribute as metadata for every other attribute in the same request. With NGSI-LD, the Standard
 `observedAt` property-of-a-property is used instead.
 
+If a `TimeInstant` arrives as measure but not follows [ISO_8601](https://en.wikipedia.org/wiki/ISO_8601) then measure
+is refused.
+
+timeInstant conf value | measure contains TimeInstant | Behaviour
```
Comment (mapedraza) [inline doc/api.md:961]:
```suggestion
is refused.

Depending on the `timeInstant` configurantion and if the measure contains a 
value named `TimeInstant` with a correct value, the IoT behaviour is described
in the following table:

timeInstant conf value | measure contains TimeInstant | Behaviour 
```

**Review by mapedraza — APPROVED** [review]
LGTM

**Pull request merged**